src/web-ui/src/mixins/product.js

```diff
@@ -28,19 +27,11 @@ export const product = {
 
       return capitalize(this.product.category);
     },
-    outOfStock() {
-      if (!this.product) return false;
-
-      return this.product.current_stock === 0;
-    },
   },
   methods: {
     async getProductByID(product_id) {
       const { data } = await ProductsRepository.getProduct(product_id);
       this.product = data;
     },
-    recordProductViewed(feature, exp, discount) {
-      AnalyticsHandler.productViewed(this.user, this.product, feature, exp, discount);
-    },
```

Comment:
I'm under the impression that we're only recording Personalize interactions during the first time login flow. 

Though I think some UI to allow the user to indicate that they liked our recommendation would be good as well. This would be another opportunity to record interactions

Reply:
Resurrected